mysite/auth.py

```diff
@@ -51,28 +51,26 @@ def login():
                 # "expiration": str(datetime.utcnow() + timedelta(seconds=120))
                 os.environ["SECRET_KEY"], algorithm="HS256")
                 return jsonify({"token": token})
-        return jsonify({})
+        return jsonify({ "message": "Login Failed. Incorrect email/password" })
 
 @auth.route("/signup", methods=["POST"])
 def signup():
     if request.method == "POST":
         full_name = request.json["full_name"]
         email = request.json["email"]
         phone_number = request.json["phone_number"]
-        password1 = request.json["password1"]
-        password2 = request.json["password2"]
-
+        password = request.json["password"]
```

Comment:
Removed password2 and refactored password1 to password because comparison and confirmation of passwords is handled on the frontend